packages/common/src/visualizations/types/index.ts

```diff
@@ -33,6 +33,12 @@ export enum VizIndexType {
     CATEGORY = 'category',
 }
 
+export function getAxisType(column: VizColumn): VizIndexType {
+    return column.type === DimensionType.STRING
+        ? VizIndexType.CATEGORY
+        : VizIndexType.TIME;
+}
```

Comment:
This should probably be an exhaustive `switch/case` of `column.type`

With this code we're returning `VizIndexType.TIME` when `column.type` is `Number` for example

Reply:
Good call

Reply:
Also is this the same as `getVizIndexTypeFromDimensionType` ?